core/lib/Thelia/TaxEngine/TaxType/FeatureFixAmountTaxType.php

```diff
@@ -46,7 +47,9 @@ public function fixAmountRetriever(Product $product)
             ->findOne();
 
         if (null !== $query) {
-            $taxAmount = $query->getFreeTextValue();
+            if ( (1 == $taxAmount = $query->getFreeTextValue()) && !is_null($query->getFeatureAvId()) ) {
```

Comment:
Its for BC. Just imagine manually filled DB (or via some import) with using of this tax type.
Seems to need add php upgrade script for converting old `free_text_values` to "default locale i18n"  `feature_avs`